docs/docs/modules.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: KAMI Blue Modules
+description: KAMI Blue Module List
+---
+
+## KAMI Blue Modules
+
+<details>
+    <summary>Capes</summary>
+    <ul>
+        <li>Capes</li>
+        <li>LayerCape</li>
+    </ul>
+</details>
+
+<details>
+    <summary>Combat</summary>
+    <ul>
+        <li>AntiDeathScreen</li>
+        <li>AntiFriendHit</li>
+        <li>Aura</li>
+        <li>Auto32k</li>
+        <li>AutoExp</li>
+        <li>AutoFeetPlace</li>
+        <li>AutoLog</li>
+        <li>AutoReplenish</li>
+        <li>AutoTotem</li>
+        <li>AutoTrap</li>
+        <li>BowSpam</li>
+        <li>Criticals</li>
+        <li>CrystalAura</li>
+        <li>Fastuse</li>
+        <li>Pull32k</li>
+        <li>Surround</li>
+    </ul>
+</details>
+
+<details>
+    <summary>Experimental</summary>
+    <ul>
+        <li>GUIColour</li>
+    </ul>
+</details>
+
+<details>
+    <summary>GUI</summary>
+    <ul>
+        <li>ArmourHide</li>
+        <li>CleanGUI</li>
+        <li>InfoOverlay</li>
+        <li>InventoryViewer</li>
+        <li>Zoom</li>
+        <li>GUIColour</li>
```

Comment:
This is in the wrong category, it's under experimental

Reply:
which modules specifically?

Reply:
ok

Reply:
The linked one. If you click view changes you can see the line it links. GUIColour is under experimental.

Reply:
oh lol i see my mistake xD

Reply:
GUIColor is now under experimental, but you left it here by accident